README.md

```diff
@@ -4,26 +4,26 @@
 
 <!-- toc -->
 
-- [Circulate](#circulate)
-  - [About](#about)
-    - [Project Considerations](#project-considerations)
-  - [Requirements](#requirements)
-  - [Integrations](#integrations)
-  - [Development](#development)
-    - [Setting up Circulate on your machine](#setting-up-circulate-on-your-machine)
-    - [Multi-tenancy](#multi-tenancy)
-    - [Configuring your database](#configuring-your-database)
-    - [Resetting the application](#resetting-the-application)
-    - [Running tests](#running-tests)
-    - [Code formatting and linting](#code-formatting-and-linting)
-    - [Setup pre-commit checks](#setup-pre-commit-checks)
-    - [Documentation](#documentation)
-    - [Who to log in as](#who-to-log-in-as)
-  - [Deployment](#deployment)
-    - [Buildpacks](#buildpacks)
-    - [Release Command](#release-command)
-    - [Daily Summary Emails](#daily-summary-emails)
-  - [Alternatives](#alternatives)
```

Comment:
This looks like it might be diff noise from separate README sprucing?

Reply:
Yeah, this is unintentional. I'll remove it.